ood-portal-generator/lib/ood_portal_generator.rb

```diff
@@ -26,5 +29,25 @@ def scl_apache?
       return false if ("#{env['ID']} #{env['ID_LIKE']}" =~ /rhel/ && env['VERSION_ID'] =~ /^8/)
       true
     end
+
+    def fqdn
+      Socket.gethostbyname(Socket.gethostname).first
+    end
+
+    def dex_user
+      Etc.getpwnam('ondemand-dex')
+      'ondemand-dex'
```

Comment:
The return value of `dex_user` is 'ondemand-dex' so `Etc.getpwnam('ondemand-dex')` would be ignored. What were you trying to achieve here?

Reply:
This does a lookup of `ondemand-dex` and if that lookup fails it returns username of person executing script.  `Etc.getpwnam` is just used to see if `ondemand-dex` user exists.